echo-terminal.md

```diff
@@ -0,0 +1,35 @@
+# Echo Terminal Snippet Web Component
+
+The Echo Terminal Snippet can be loaded as a web component via JSDelivr CDN.
+
+The JSDelivr URLs below point to the latest commit in the main branch, ensuring changes to the snippet are automatically deployed to consuming websites.
+
+[See here for more options when testing using JSDelivr](https://www.jsdelivr.com/).
+
+## Usage
+
+The Echo terminal can be loaded via both JS and HTML.
+
+### Loading with Javascript (preferred)
```

Comment:
❓ I am curious, why is this method over a `<script/>`?

Reply:
From experience, this tends to be the more common embed option I see across CMS tools, so I've recommended it as preferred to align with this commonality. There's no technical reason for this preference.